Update scripting-fundamentals.md (#306)
OfficeDev/office-scripts-docs: docs/develop/scripting-fundamentals.md

Ranges have three core properties: values, formulas, and format. These properties get or set the cell values, formulas to be evaluated, and the visual formatting of the cells. They are accessed through `getValues`, `getFormulas`, and `getFormat`. Values and formulas can be changed with `setValues` and `setFormulas`, while the format is a `RangeFormat` object comprised of several smaller objects that are individually set.

> Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section](javascript-objects.md) of [Built-in JavaScript objects in Office Scripts](javascript-objects.md#working-with-ranges) for more information on handling those arrays in the Office Scripts framework.

Reviewer: @lindalu-MSFT, what's the preferred way to link to an article section?

Reviewer 2: To link to a bookmark in a different article, I'd do this:

For more information on handling arrays in the Office Scripts framework, see [Working with ranges](javascript-objects.md#working-with-ranges). 

It's not necessary to include the link to the source topic, unless you think it will add further clarity. Keep it simple.

> Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section](javascript-objects.md) of [Built-in JavaScript objects in Office Scripts](javascript-objects.md#working-with-ranges) for more information on handling those arrays in the Office Scripts framework.

Reviewer: Also, I think these should be flipped in your PR.
```suggestion
Ranges use two-dimensional arrays to manage information. Read the [Working with ranges section](javascript-objects.md#working-with-ranges) of [Built-in JavaScript objects in Office Scripts](javascript-objects.md) for more information on handling those arrays in the Office Scripts framework.
```

Reviewer 2: Yup just saw that!

When an Excel object has a collection of one or more objects of the same type, it stores them in an array. For example, a `Workbook` object contains a `Worksheet[]`. This array is accessed by the `Workbook.getWorksheets()` method. `get` methods that are plural, such as `Worksheet.getCharts()`, return the entire object collection as an array. You'll see this pattern throughout the Office Scripts APIs: the `Worksheet` object has a `getTables()` method that returns a `Table[]`, the `Table` object has a `getColumns()` method that returns a `TableColumn[]`, as so on.

> The returned array is a normal array, so all the regular array operations are available for your script. You can also access individual objects within the collection using the array index value. For example, `workbook.getTables()[0]` returns the first table in the collection. Read the [Working with collections section](javascript-objects.md#working-with-collections) of (Built-in JavaScript objects in Office Scripts](javascript-objects.md) to learn more about using built-in array functionality with the Office Scripts framework.

Reviewer: ```suggestion
The returned array is a normal array, so all the regular array operations are available for your script. You can also access individual objects within the collection using the array index value. For example, `workbook.getTables()[0]` returns the first table in the collection. For more information on using the built-in array functionality with the Office Scripts framework, see [Work with collections](javascript-objects.md#work-with-collections). 
```
